MvvmCross/Platforms/Wpf/Core/MvxWpfSetup.cs

```diff
@@ -97,10 +91,14 @@ protected override IMvxNameMapping CreateViewToViewModelNaming()
             return new MvxPostfixAwareViewToViewModelNameMapping("View", "Control");
         }
 
+        protected override void InitializeFirstChance()
+        {
+            RegisterPresenter();
+        }
+
         protected override void InitializeLastChance()
         {
             InitializeBindingBuilder();
-            base.InitializeLastChance();
```

Comment:
Why are we removing this call? It's good practice to call the base method, even though currently the base method doesn't do anything.

Reply:
@nickrandolph my mistake - this was an oversight when I was removing some test code I must have removed the base calls as well - its fixed now.